src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1601,12 +1709,1043 @@ namespace Google.Example.Library.V1.Samples
+        /// the description into multiple lines and the samples will be just fine
+        /// </summary>
+        public static void SampleGetBook()
+        {
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            GetBookRequest request = new GetBookRequest
+            {
+                BookName = new BookName("[SHELF_ID]", "[BOOK_ID]"),
+            };
+            Book response = libraryServiceClient.GetBook(request);
+            string intKeyVal = response.MapStringValue[123];
+            string booleanKeyVal = response.MapBoolKey[true];
+            int mapValueField = response.MapMessageValue["key"].Field;
+            Console.WriteLine($"Test printing map fields: {response.MapListValueValue["quoted_key"]}");
+            Console.WriteLine($"Test printing enum fields of a map value: {response.MapMessageValue["key"].Alignment}");
+        }
+        // [END sample]
+        public static void Main(string[] args)
+        {
+            SampleGetBook();
+        }
+    }
+}
+============== file: Google.Example.Library.V1/Google.Example.Library.V1.Samples/Lovelace.cs ==============
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("RequestStreamingServer", "prog")
+
+// sample-metadata
+//   title: Programming Books
+//   description: Testing calling forms
+//   usage: dotnet run
+
+using CommandLine;
+
+// [START lovelace]
+namespace Google.Example.Library.V1.Samples
+{
+    using Google.Example.Library.V1;
+    using System;
+
+    public class Lovelace
+    {
+        /// <summary>
+        /// Testing calling forms
+        /// </summary>
+        public static async Task SampleStreamBooksAsync()
+        {
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            StreamBooksRequest request = new StreamBooksRequest
+            {
+                Name = "BASIC",
+            };
+
+            // Make the request, returning a streaming response
+            LibraryServiceClient.StreamBooksStream streamingResponse =
+                AsyncServerStreamingCall<Book> response = libraryServiceClient.StreamBooks(request);
+
+            // Read streaming responses from server until complete
+            IAsyncEnumerator<Book> responseStream = streamingResponse.ResponseStream;
+            while (await responseStream.MoveNext())
```

Comment:
Alternatively use the `ForEachAsync` method:
```C#
await responseStream.ForEachAsync(item =>
{
    Console.WriteLine(item);
});
```

Reply:
Done